src/rgw/rgw_common.h

```diff
@@ -1693,6 +1693,7 @@ struct req_state {
   bool enable_usage_log;
   uint8_t defer_to_bucket_acls;
   uint32_t perm_mask;
+  string subuser_name;
```

Comment:
@rzarzynski  sorry, I've encountered some compile issue with `boost::string_ref`, will change `std::string` to `boost::string_ref`  later.